yarn.lock

```diff
@@ -2494,7 +2494,7 @@ dompurify@^2.2.6:
   version "2.5.6"
   resolved "https://registry.yarnpkg.com/dompurify/-/dompurify-2.5.6.tgz#8402b501611eaa7fb3786072297fcbe2787f8592"
   integrity sha512-zUTaUBO8pY4+iJMPE1B9XlO2tXVYIcEA4SNGtvDELzTSCQO7RzH+j7S180BmhmJId78lqGU2z19vgVx2Sxs/PQ==
-
+  
```

Comment:
Hi @ishitasequeira , could you try undoing the changes in your workspace, and only change line 69 of the package.json file, to 5.94.0. Then run `yarn install`, and that should update the yarn.lock file automatically.